builder/tealium-swift.xcodeproj/project.pbxproj

```diff
@@ -9361,6 +9385,7 @@
 		CFAD9D3325A66BE90079B19A /* Debug */ = {
 			isa = XCBuildConfiguration;
 			buildSettings = {
+				ALWAYS_EMBED_SWIFT_STANDARD_LIBRARIES = NO;
```

Comment:
Is this the fix for that issue that Wells Fargo brought to us? Not sure this is where this should go - though its probably okay it just needs documented somewhere that we made this change here.

Reply:
@trister1997 do you mean it shouldn't be in this PR? i just didn't have a task for this minor fix, i can cherry pick this commit to the release branch so we'll have a separate commit for this on main